libs/ledger-live-common/src/reconciliation.ts

```diff
@@ -29,8 +27,26 @@ import {
   fromSolanaResourcesRaw,
   fromNFTRaw,
   fromCardanoResourceRaw,
+  toCryptoOrgResourcesRaw,
+  toSolanaResourcesRaw,
+  toCosmosResourcesRaw,
+  toTronResourcesRaw,
 } from "./account";
 import consoleWarnExpectToEqual from "./consoleWarnExpectToEqual";
+import { BitcoinAccount, BitcoinAccountRaw } from "./families/bitcoin/types";
+import { TronAccount, TronAccountRaw } from "./families/tron/types";
+import { CosmosAccount, CosmosAccountRaw } from "./families/cosmos/types";
+import { AlgorandAccount, AlgorandAccountRaw } from "./families/algorand/types";
+import _ from "lodash";
+import { PolkadotAccount, PolkadotAccountRaw } from "./families/polkadot/types";
```

Comment:
Unsual import order :stuck_out_tongue:

Reply:
Indeed ! And I even managed to remove the lodash import, isEqual is already imported higher in the list